types/greenlock-express/index.d.ts

```diff
@@ -0,0 +1,28 @@
+// Type definitions for greenlock-express 4.0
+// Project: https://git.rootprojects.org/root/greenlock-express.js.git
+// Definitions by: Joseph Hendrix <https://github.com/hendrixjoseph>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { RequestListener } from 'http';
+
+export interface opts {
+    packageRoot: string;
+    maintainerEmail: string;
+    configDir: string;
+    cluster: boolean;
+    workers?: number;
+}
+
+export interface serve {
+    ready: (app?: RequestListener) => serve;
+    master: (app?: RequestListener) => serve;
+    serve: (app: RequestListener) => void;
+}
+
+export interface greenlock {
+    init: (o: (opts | (() => opts))) => serve;
+}
+
+declare const gl: greenlock;
+
+export default gl;
```

Comment:
```suggestion
export = gl;
```
https://git.rootprojects.org/root/greenlock-express.js/src/branch/master/greenlock-express.js#L7
this looks like CJS module, old school Node IMO

Reply:
yeah, it's pretty old (4 years since the last release) but it still seems to be recommended on stack overflow.

when I changed the default export to just `=` I had to remove the other exports:

`An export assignment cannot be used in a module with other exported elements. ts(2309)`

and had to change the import in the test to:

```typescript
import greenlock = require('greenlock-express');
```

which is fine. I'm having a little trouble with your other PR comment; I'll add in the discussion thread there.